app/controllers/concerns/remote_record_controller.rb

```diff
@@ -0,0 +1,53 @@
+module RemoteRecordController
+  extend ActiveSupport::Concern
+
+  included do
+    def curation_concern
+      if wants_to_update_remote_metadata?
+        decorated_concern
+      else
+        @curation_concern
+      end
+    end
+
+    def manifest
+      respond_to do |f|
+        f.json do
+          render json: manifest_builder
+        end
+      end
+    end
+
+    private
+
+    def presenter
+      @presenter ||=
+        begin
+          _, document_list = search_results(params, CatalogController.search_params_logic)
+          curation_concern = document_list.first
+          raise CanCan::AccessDenied.new(nil, :manifest) unless curation_concern
+          @presenter = show_presenter.new(curation_concern, current_ability)
+        end
+    end
+
+    def manifest_builder
+      ManifestBuilder.new(presenter)
+    end
+
+    def show_presenter
+      ScannedResourceShowPresenter
```

Comment:
No -- that should be moved out to the controllers.  Right now they're basically the same, but that'll change as UI for MultiVolumeWork's members get fleshed out.